examples/typescript/quickstart/tsconfig.json

```
@@ -0,0 +1,14 @@
+{
+  "compilerOptions": {
+    "target": "ES2020",
+    "module": "ESNext",
+    "moduleResolution": "Node",
```

The value "Node" for `moduleResolution` is outdated
```suggestion
    "moduleResolution": "NodeNext",
```